addons/website_slides_survey/security/website_slides_survey_security.xml

```diff
@@ -1,76 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<odoo>
-    <data noupdate="1">
-    <!-- Specific survey access rules for slide channel officer-->
-    <record id="survey_user_input_rule_slide_channel_officer" model="ir.rule">
-        <field name="name">Survey user input: slide channel officer on certification: read</field>
-        <field name="model_id" ref="survey.model_survey_user_input"/>
-        <field name="domain_force">[('survey_id.certification', '=', True),
-            ('survey_id.survey_type', 'in', ('survey', 'live_session', 'assessment', 'custom')),
-            '|', ('survey_id.restrict_user_ids', '=', False), ('survey_id.restrict_user_ids', 'in', user.id)]</field>
-        <field name="groups" eval="[(4, ref('website_slides.group_website_slides_officer'))]"/>
-        <field name="perm_unlink" eval="0"/>
-        <field name="perm_write" eval="0"/>
-        <field name="perm_read" eval="1"/>
-        <field name="perm_create" eval="0"/>
```

Comment:
Same remarque as for hr_recruitment, I see here that we only grant officer the right to read those survey/input/line and I fail to see where this is implemented now

Reply:
see https://github.com/odoo/odoo/pull/157348#discussion_r1591960112